lib/inspec/base_cli.rb

```diff
@@ -60,6 +60,8 @@ def self.exec_options
         desc: 'Load attributes file (experimental)'
       option :cache, type: :string,
         desc: 'Use the given path for caching dependencies. (default: ~/.inspec/cache)'
+      option :no_write_lockfile, type: :boolean, default: false,
```

Comment:
maybe `create-lockfile`, since otherwise I am thinking its overwriting the old lockfile